werkzeug/wrappers.py

```diff
@@ -1434,6 +1458,23 @@ def make_conditional(self, request_or_environ):
         :param request_or_environ: a request object or WSGI environment to be
                                    used to make the response conditional
                                    against.
+        :param accept_ranges: This parameter dictates the value of
+                              `Accept-Ranges` header. If ``False`` (default),
+                              the header is not set. If ``True``, it will be set
+                              to ``"bytes"``. If ``None``, it will be set to
+                              ``"none"``. If it's a string, it will use this
+                              value.
+        :param complete_length: Will be used only in valid Range Requests.
+                                It will set `Content-Range` complete length
+                                value and compute `Content-Length` real value.
+                                This parameter is mandatory for successful
+                                Range Requests completion.
+        :param ignore_invalid_range_requests: set to ``True`` to silently degrade
+                                              to complete HTTP response for
+                                              invalid range requests, instead of
+                                              raising an HTTP 406 exception.
```

Comment:
> An origin server MUST ignore a Range header field that contains a range unit it does not understand.

This refers to incoming client `Range` header, we should still be able to generate whatever we want for `Accept-Ranges` value.

I'm looking how to impact properly the 2 other points.